README.md

```diff
@@ -33,3 +33,9 @@ OpenMRS server is accessible at http://localhost/openmrs
 ### Known limitations
 
 - Only OpenMRS Component is now supported.
+
+**export the variable beforehand:**
```

Comment:
@Fabiencha this should not be added just like that at the end of the file.
`docker-compose up` is already in there. And there is already a section to add instructions.